### PR TITLE
Test coverage   

### PR DESCRIPTION
**Pull request by benceHornyak**
Please give me a review about the test.. Are there any bad-practice that I used? Any way to improve the test?

Thanks

part of #2

### CODE REVIEW

**Comment by Ethan-Arrowood** [comment]
This looks good so far! Keep it up 😄

**Comment by benceHornyak** [comment]
I had to make some changes regarding the scope checking to be 100%.. included a happy path as well.. 
Without it line 122 wouldn't throw the error

**Comment by Ethan-Arrowood** [comment]
Incredible! Great job. Let me do a quick code review and lets see if we can get this merged.

File: test.js [hunk 1]
```diff
@@ -26,4 +26,144 @@ test('should decorate request instance with jwtAuthz method', function (t) {
...
+      uri: 'http://localhost:' + fastify.server.address().port + '/test5',
+      json: true
+    }, function (err, response) {
+      t.error(err)
+      t.ok(response)
+    })
+  })
+})
+
+test('Happy path', function (t) {
```
Comment (Ethan-Arrowood) [inline test.js]: lets rename this test to 'should verify user scope'

**Review by Ethan-Arrowood — CHANGES_REQUESTED** [review]
Okay just change the name of the happy path test and its good to merge 😄

**Review by Ethan-Arrowood — APPROVED** [review]
LGTM 🎉

**Comment by Ethan-Arrowood** [comment]
Great work! Once Travis-CI passes i'll merge this. Thank you for contributing 👍

**Pull request merged**